client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -200,6 +201,47 @@ class ShippingLabels extends Component {
 		);
 	};
 
+	renderEmailReceiptsSection = () => {
+		const {
+			emailReceipts,
+			translate,
+			masterUserName,
+			masterUserLogin,
+			canEditSettings,
+			canEditPayments,
+			setFormDataValue,
+		} = this.props;
+
+		if ( ! isBoolean( emailReceipts ) ) {
+			return null;
+		}
+
+		const onChange = () => setFormDataValue( 'email_receipts', ! emailReceipts );
+
+		return (
+			<FormFieldSet>
+				<FormLabel className="label-settings__cards-label">
+					{ translate( 'Email Receipts' ) }
+				</FormLabel>
+				<FormLabel>
+					<FormCheckbox
+						checked={ emailReceipts }
+						onChange={ onChange }
+						disabled={ ! canEditPayments && ! canEditSettings }
+					/>
+					<span className="label-settings__credit-card-description">
+						{ translate( 'Email the label purchase receipts to %(ownerName)s (%(ownerLogin)s)', {
```

Comment:
Seems like the setting should include displaying the email address. As a user, I would have to go check that username's profile to see where emails are actually going. So, instead could we do something like "Email label purchase receipts to {displayname} ({username}) at {useremail}"?

cc @kellychoffman @jameskoster

Reply:
Great point. I agree.